mobile/library/objective-c/EnvoyEngineImpl.mm

```diff
@@ -530,8 +530,7 @@ - (int)runWithConfig:(EnvoyConfiguration *)config logLevel:(NSString *)logLevel
     options->setConcurrency(1);
     return reinterpret_cast<Envoy::Engine *>(_engineHandle)->run(std::move(options));
   } @catch (NSException *exception) {
-    NSLog(@"[Envoy] exception caught: %@", exception);
-    [NSNotificationCenter.defaultCenter postNotificationName:@"EnvoyError" object:self];
```

Comment:
Alternatively on the library consumer side, we could observe these notifications and react however we like (emit an internal error at the app level). We'd probably want to attach the exception itself to the notification though:

```objc
[NSNotificationCenter.defaultCenter postNotificationName:@"EnvoyError" object:exception];
```

Reply:
I improved the notification per ur suggestion but kept logging the way it was.